client/src/components/Button/Button.stories.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import Button from "./Button";
+import "./Button.css";
+
+storiesOf("Button", module)
+  .add("light-pink", () => <Button label="Book Now" type="light-pink" />)
```

Comment:
Can you make these labels more human-friendly? Like `Light Pink`, `Dark Pink`, `Red`, etc